src/V3Broken.cpp

```diff
@@ -175,8 +175,11 @@ class BrokenCheckVisitor final : public VNVisitorConst {
     void processEnter(AstNode* nodep) {
         nodep->brokenState(m_brokenCntCurrentUnder);
         const char* const whyp = nodep->broken();
-        UASSERT_OBJ(!whyp, nodep,
-                    "Broken link in node (or something without maybePointedTo): " << whyp);
+        if (whyp) {
+            UINFO(0, "In node: " << nodep << endl);
+            nodep->v3fatalSrc(
+                "Broken link in node (or something without maybePointedTo): " << whyp);
+        }
```

Comment:
Please revert.  With --debug it will already print the node for this or any other error, thanks.

Reply:
Oh I didn't noticed the detailed node info is printed when --debug is on, sorry for that...